opensrs/opensrsapi.py

```diff
@@ -443,11 +444,10 @@ def renew_domain(self, domain, current_expiration_year, period):
             rsp = self._renew_domain(domain, current_expiration_year, period)
             return rsp.get_data()['attributes']['order_id']
         except errors.XCPError, e:
-            # We cannot renew .de domains right now (they are
-            # auto-renewed automatically on registar side).
-            # Therefore consider each .de domain renewal
-            # attempt as successful
-            if self._already_renewed(e) or self._is_de_domain_case(e, domain):
+            # We cannot control domains which are automatically renewed on
+            # OpenSRS side. Thus we always treat them as already renewed
+            # for each renewal attempt.
+            if self._already_renewed(e) or self._is_auto_renewed(e, domain):
```

Comment:
It's a hard question (at least for me) and I dunno the answer for it :(.  We don't have many auto-renewed domains, so, intuitively I suppose `_already_renewed()` will be happening more often.